packages/core/package.json

```diff
@@ -87,6 +88,7 @@
     "react-hot-loader": "^4.12.21",
     "react-oembed-container": "^1.0.0",
     "react-redux": "^7.2.0",
+    "react-tracking": "^8.0.0",
```

Comment:
This will be removed after we have the `@irvingjs/tracking` package setup and working right to replace the tracking service